package.json

```diff
@@ -8,7 +8,10 @@
     "clockout": "node execute.js clockout",
     "scheduler:start": "node start.js",
     "scheduler:stop": "node stop.js",
-    "test-fix": "node test-script/test-fix.js"
+    "test-fix": "node test-script/test-fix.js",
```

Comment:
can you add script that will run all `test-*` file in `test-script/` folder too?